### ISSUE TITLE
Tiny yolo: adding new class - loss is nan

### ISSUE THREAD

**Issue by AGrosserHH** [qqwweee/keras-yolo3#450]
Hi all,

I have added a new class to tiny yolo and try to train the model. When I train I only get the information that the loss is nan.
![image](https://user-images.githubusercontent.com/11669082/57681677-b12b9d00-761f-11e9-987a-6193f3f32de7.png)

Is there any solution?

Cheers, 
Andi

**Comment by EnesBasarkan** [comment]
Did you solve the problem?

**Comment by datduonguva** [comment]
I posted my solution in here. Hope it would help.
https://github.com/qqwweee/keras-yolo3/issues/171#issuecomment-522287613
